robolectric/src/main/java/org/robolectric/internal/bytecode/InstrumentationConfiguration.java

```diff
@@ -187,22 +209,13 @@ public boolean shouldAcquire(String name) {
     // See https://github.com/robolectric/robolectric/issues/521
     if (name.equals("android.R$styleable")) return true;
 
-    return !(
-        name.matches(".*\\.R(|\\$[a-z]+)$")
-            || classesToNotAquire.contains(name)
-            || name.startsWith("java.")
-            || name.startsWith("javax.")
-            || name.startsWith("sun.")
-            || name.startsWith("com.sun.")
-            || name.startsWith("org.w3c.")
-            || name.startsWith("org.xml.")
-            || name.startsWith("org.junit")
-            || name.startsWith("org.hamcrest")
-            || name.startsWith("org.specs2")  // allows for android projects with mixed scala\java tests to be
-            || name.startsWith("scala.")      //  run with Maven Surefire (see the RoboSpecs project on github)
-            || name.startsWith("kotlin.")
-            || name.startsWith("com.almworks.sqlite4java") // Fix #958: SQLite native library must be loaded once.
-    );
+    if (packagesToNotAquire.contains(pkgName)) return false;
```

Comment:
> I can change it to simpler version if we don't care for performance so much in this place.

I think the previous implementation was easier to read. If someone else comes into this code in the future, its not obvious that it's implemented the way it is for performance reasons. My main concern, however, is that we preserve the existing behavior.